### ISSUE TITLE
[MAS3.2.1][Keyboard Navigation Services->+symbol] When the user press the enter key on Services plus button focus is not on the menu item 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1764]
**Actual Result:**
When the user presses the enter key on Services plus button focus is not on the menu item
Narrator focus also not moving to the menu item 
When the user cancels the menu item using esc key , focus is not lands on the Services plus button 

**Expected Result:**
When the user presses the enter key on Services plus button focus should be lands on  the “Add Language understanding(LUIS) only and should read the menu item information 
When the user cancels the menu item using esc key , focus should be lands on the Services plus button only.

**Repro Steps:**
1.	Open the Bot Framework V4 Emulator.
2.	Navigate to the Bot explorer and select it 
3.	Navigate to the service plus button and select it 
4.	Observe the issue

**User Impact:**
Users who rely on keyboard navigation will be affected if the keyboard focus is not lands on  the expected control keyboard users will be affected where the exact keyboard focus lands on it . 

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator

**MAS Reference Link:**
[Sharepoint Link](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a4bbd37d-ee11-485e-b5c8-0539f0ac94b2})
[MAS3.2.1_Add Language menu.zip](https://github.com/microsoft/BotFramework-Emulator/files/3529482/MAS3.2.1_Add.Language.menu.zip)
[MAS3.2.1_Add Language.zip](https://github.com/microsoft/BotFramework-Emulator/files/3529484/MAS3.2.1_Add.Language.zip)

**Comment by corinagum** [comment]
![image](https://user-images.githubusercontent.com/14900841/63546596-07da2a00-c4df-11e9-9c29-6e0a62afdba2.png)

Looks like the context menu is trying to focus where the keyboard is, rather than where the + button is.

**Comment by tonyanziano** [comment]
Not sure what we can do about this one in the short term. The context menu is native to the OS, and there is no API available allowing us to control what option gets focused when it is opened.

To resolve this we would have to write a custom control.

**Comment by corinagum** [comment]
This is related to the context menu. Postponing for now.

**Comment by v-rampul** [comment]
#HCL-BotFrameworkEmulator; #Emulator-Win; #A11yMAS; #HCL; #MAS3.2.1; #A11ySev3
